src/api/kube.ts

```diff
@@ -57,6 +57,25 @@ export class KubeHelper {
     }
   }
 
+  async createNamespace(namespaceName: string, labels: any): Promise<void> {
```

Comment:
I think earlier we used https://github.com/che-incubator/chectl/blob/master/src/tasks/installers/common-tasks.ts#L24 . @tolusha  @flacatus maybe we should remove this method and use KubeHelper#createNamespace ? I don't think that we should have duplicated methods.

Reply:
Also we could make labels like optional arg.